static/js/common/init.js

```diff
@@ -5,11 +5,11 @@ MEDUSA.common.init = function() {
     if (MEDUSA.config.fanartBackground) {
         var showID = $('#showID').attr('value');
         if (showID) {
-            let asset = 'show/' + $('#showID').attr('value') + '?type=fanart';
+            let asset = 'show/' + showID + '?type=fanart';
             let path = apiRoot + 'asset/' + asset + '&api_key=' + apiKey;
             $.backstretch(path);
-            $('.backstretch').css('top', backstretchOffset());
-            $('.backstretch').css('opacity', MEDUSA.config.fanartBackgroundOpacity).fadeIn(500);
+            $('.backstretch')
```

Comment:
Why is the on a seperate line? Js doesn't have the line length that Python does. Just make it into a long line.